.github/workflows/release.yaml

```diff
@@ -45,4 +45,14 @@ jobs:
           EG_TAG_VERSION: ${{ steps.tag_env.outputs.version}}
         run: make image.push.multiarch TAG=${EG_TAG_VERSION} IMAGE_NAME=gateway PLATFORMS="linux_amd64 linux_arm64"
 
+      - name: Generate Release Manifests
+        run: make release-manifests
+
+      - name: Upload Release Manifests
+        uses: svenstaro/upload-release-action@v2
```

Comment:
is this an alternative to create a release and uploading a release asset using the standard actions ?
https://github.com/actions/create-release#example-workflow---create-a-release

Reply:
I will take a look and figure it out, I used this action before and it worked well:)

Reply:
@arkodg the official `create-release` action is unmaintained and not recommended: https://github.com/actions/create-release#github-action---releases-api

Reply:
I am going to try this https://github.com/softprops/action-gh-release in recommendation list.